Cargo.toml

```diff
@@ -79,7 +79,7 @@ sharded-slab = "0.1.1"
 strsim = "0.10"
 tar = "0.4.26"
 tempfile.workspace = true
-term = "=0.5.1" # FIXME(issue #1818, #1826, and friends)
+termcolor = "1.2"
```

Comment:
please add this to the 'workspace dependencies' table below, and this line here should be
```
termcolor.workspace = true
```

This will help us keep a single version of the dependency in our dependencies even if we refactor rustup in future.